src/frontend/Komponenter/Behandling/Simulering/Tilbakekreving.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useEffect, useState } from 'react';
+import { Radio } from 'nav-frontend-skjema';
+import { EnsligTextArea } from '../../../Felles/Input/TekstInput/EnsligTextArea';
+import { FamilieRadioGruppe } from '@navikt/familie-form-elements';
+import { Hovedknapp } from 'nav-frontend-knapper';
+import Søknad from '../../../Felles/Ikoner/Søknad';
+import IkonKnapp from '../../../Felles/Knapper/IkonKnapp';
+import styled from 'styled-components';
+import { useApp } from '../../../App/context/AppContext';
+import { Ressurs, RessursStatus } from '../../../App/typer/ressurs';
+import { useHistory, useParams } from 'react-router-dom';
+import { IBehandlingParams } from '../../../App/typer/routing';
+import { AlertStripeFeil } from 'nav-frontend-alertstriper';
+import { useBehandling } from '../../../App/context/BehandlingContext';
+
+enum ITilbakekrevingsvalg {
+    OPPRETT_MED_VARSEL = 'OPPRETT_MED_VARSEL',
+    OPPRETT_UTEN_VARSEL = 'OPPRETT_UTEN_VARSEL',
+    AVVENT = 'AVVENT',
+}
+
+interface ITilbakekreving {
+    valg?: ITilbakekrevingsvalg;
+    varseltekst?: string;
+    begrunnelse: string;
+}
+
+const VarselValg = styled.div`
+    margin-bottom: 1rem;
+`;
+
+export const Tilbakekreving: React.FC = () => {
+    const { nullstillIkkePersisterteKomponenter, settIkkePersistertKomponent } = useBehandling();
+    const { axiosRequest } = useApp();
+    const { behandlingId } = useParams<IBehandlingParams>();
+    const history = useHistory();
+    const [tilbakekrevingsvalg, settTilbakekrevingsvalg] = useState<ITilbakekrevingsvalg>();
+    const [varseltekst, settVarseltekst] = useState<string>('');
+    const [begrunnelse, settBegrunnelse] = useState<string>('');
+    const [feilmelding, settFeilmelding] = useState<string>();
+    const [låsKnapp, settLåsKnapp] = useState<boolean>(false);
+
+    // TODO: HarÅpenTilbakekreving - vis info om dette istedenfor skjema
+
+    useEffect(() => {
+        axiosRequest<ITilbakekreving, null>({
+            method: 'GET',
+            url: `familie-ef-sak/api/tilbakekreving/${behandlingId}`,
+        }).then((respons: Ressurs<ITilbakekreving>) => {
+            if (respons.status === RessursStatus.SUKSESS && respons.data) {
+                settBegrunnelse(respons.data.begrunnelse);
+                settTilbakekrevingsvalg(respons.data.valg);
+                settVarseltekst(respons.data.varseltekst || '');
+            } else if (
+                respons.status === RessursStatus.FEILET ||
+                respons.status === RessursStatus.FUNKSJONELL_FEIL ||
+                respons.status === RessursStatus.IKKE_TILGANG
+            ) {
+                settFeilmelding(respons.frontendFeilmelding);
+            }
+        });
+        // eslint-disable-next-line
+    }, []);
```

Comment:
Burde kanskje ha med denne
```suggestion
    }, [behandlingId]);
```

Reply:
Ja, kan funke det. Skal jo "i teorien" ikke kunne bytte behandlingId uten at siden relaster, men grei sikkerhet iallefall!